instances/tissueSampleType/homogeneousCellPopulation.jsonld

```diff
@@ -2,13 +2,13 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/tissueSampleType/cellCulture",
+  "@id": "https://openminds.ebrains.eu/instances/tissueSampleType/homogeneousCellPopulation",
   "@type": "https://openminds.ebrains.eu/controlledTerms/TissueSampleType",
-  "definition": "Cells taken from a living organism and grown under controlled conditions (in culture).",
+  "definition": "A sample of multiple cells/a population of cells that are of the same cell type.",
   "description": null,
   "interlexIdentifier": null,
   "knowledgeSpaceLink": null,
-  "name": "cell culture",
-  "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/BTO_0000214",
+  "name": "homogeneous cell population",
+  "preferredOntologyIdentifier": null,
   "synonym": null
```

Comment:
maybe here you should mention: cell line ?

Reply:
I wasn't sure about that. With broadening the name to homogeneous cell population, I don't think "cell line" would be a synonym anymore, but rather an example of such a population. What do you think?

Reply:
not sure. leave it out